docs/configuration/elements.md

```diff
@@ -18,7 +18,7 @@ Namespace: `options.elements.point`, global point options: `Chart.defaults.eleme
 
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
-| `radius` | `number` | `3` | Point radius.
+| `radius` | `number`\|`number[]` | `3` | Point radius.
```

Comment:
All the properties are indexable so you can also add this to the other props

Reply:
@LeeLenaleee Can you clarify what this means? I would be saddened if a documentation update held up the 4.5.0 release